app/api/v1/entities/shallow.rb

```diff
@@ -35,6 +39,7 @@ class ProjectRoleShallow < Shallow
 
   class ProtocolShallow < Shallow
     root 'protocols', 'protocol'
+    expose :short_title
```

Comment:
Shallow class object attributes should be limited to [:id, :callback_url]. Slippery slope